app/src/main/java/co/nimblehq/compose/crypto/ui/navigation/ComposeCryptoApp.kt

```diff
@@ -0,0 +1,45 @@
+package co.nimblehq.compose.crypto.ui.navigation
+
+import android.widget.Toast
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalContext
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import co.nimblehq.compose.crypto.CryptoAppState
+import co.nimblehq.compose.crypto.extension.collectAsEffect
+
+@Composable
+fun ComposeCryptoApp(
+    navController: NavHostController = rememberNavController(),
+    cryptoAppState: CryptoAppState
+) {
+    var onInternetRestore: () -> Unit = {}
```

Comment:
How about renaming it to `callback`?
```suggestion
    var callback: () -> Unit = {}
```